site/content/docs/v1.9/_index.md

```diff
@@ -33,7 +33,7 @@ If you encounter issues, review the [troubleshooting docs][30], [file an issue][
 
 ## Contributing
 
-If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.9/start-contributing/) documentation for guidance on how to setup Velero for development.
+If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.9.0/start-contributing/) documentation for guidance on how to setup Velero for development.
```

Comment:
Not sure about that.
On velero.io, recent releases use v1.x as branch name.

Reply:
yeah, I will modify it